Tools/MiniBrowser/wpe/CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ endif ()
 
 if (DEVELOPER_MODE)
     list(APPEND MiniBrowser_PRIVATE_DEFINITIONS WEBKIT_INJECTED_BUNDLE_PATH="${CMAKE_LIBRARY_OUTPUT_DIRECTORY}")
+    file(COPY "${CMAKE_BINARY_DIR}/WebInspectorUI/DerivedSources/inspector.gresource" DESTINATION "${CMAKE_BINARY_DIR}/share")
```

Comment:
I suspect this should be placed elsewhere. I'll let Adrian chime in :)

Reply:
Indeed, this would belong in `Source/WebKit/PlatformWPE.cmake`. But I would rather avoid copying the file.